tests/configurable_cluster.py

```diff
@@ -194,29 +208,46 @@ def copy_to_host(self, source_path, host, dest_path=None):
             dest_path = os.path.join(self.mount_dir,
                                      os.path.basename(source_path))
         self.exec_cmd_on_host(host, 'mkdir -p ' + os.path.dirname(dest_path))
+        self.exec_cmd_on_host(host, 'chown {0}:{0} {1}'.format(self.user, os.path.dirname(dest_path)))
```

Comment:
Prefer dict-based formatting to make it less likely that somebody adds a parameter and screws up the positions.